include/Geometry.h

```diff
@@ -4,16 +4,18 @@
 #include <glm/glm.hpp>
 #include <GLFW/glfw3.h>
 
-struct Geometry
+class Geometry
```

Comment:
Let's call everything a class and just add accessibility modifiers to make encapsulation explicit